### DISCUSSION TITLE
CORS Support

### DISCUSSION THREAD

**Discussion by gautamkct** [OpenIDC/mod_auth_openidc#729] [Q&A] [answered]
Hi All,

Would like to check if CORS is by default supported by mod_auth_openidc  or is there any specific option that needs to be enabled to allow access from other domains.

Thanks

**Comment by zandbelt** [comment, marked as answer]
Apache can be configured for CORS independently of mod_auth_openidc, see https://stackoverflow.com/questions/29150384/how-to-allow-cross-domain-request-in-apache2

**Comment by gautamkct** [comment]
Thanks @zandbelt  for prompt response. Since  you confirmed Apache can be configured for CORS independently of mod_auth_openidc, We should be good now.
